guides/advanced_guides/search_parameters.md

```diff
@@ -66,6 +68,192 @@ If you want to get only **two** documents, set `limit` to `2`.
 
 <code-samples id="search_parameter_guide_limit_1" />
 
+## Filters
+
+`filters=<String>`
+
+Specify a filter to be used with the query. See our [dedicated guide](/guides/advanced_guides/filtering.md).
+
+<code-samples id="search_parameter_guide_filter_1" />
+
+```json
+{
+  "id": "569367",
+  "title": "Nightshift",
+  "poster": "https://image.tmdb.org/t/p/w1280/peOeFl8ZTBTCERz5XQZAjYbXYsQ.jpg",
+  "overview": "Amy begins her first night shift in a hotel with a murderous past. Witnessing terrifying events and trapped within a loop, Amy must find a way to escape the flesh obsessed murderer and save residents of the hotel.",
+  "release_date": 1536282000
+}
+```
+
+The parameter should be **URL-encoded**.
+
+<code-samples id="search_parameter_guide_filter_2" />
+
+## Facet filters
+
+If you have [set up faceted attributes](/guides/advanced_guides/settings.md#attributes-for-faceting), you can filter on [facets](/guides/advanced_guides/faceted_search.md) to narrow down your results based on criteria.
+
+`facetFilters=["facetName:facetValue"]` or `facetFilters=[["facetName:facetValue"]]`
+or a mix of both `facetFilters=["facetName1:facetValue1", ["facetName2:facetValue2"]]`
+
+- `["facetName1:facetValue1", ["facetName2:facetValue2"]]` (Array of array of strings or single strings, defaults to `null`)
+
+  Both types of array contain the facet names and values to filter on.
+  A valid array must be an array which contains either a list of strings or arrays of strings and can mix both (e.g. `["director:Mati Diop", ["genre:Comedy", "genre:Romance"]]`).
```

Comment:
which  is for non-defining clause, when it is essential, use that (https://www.grammarly.com/blog/which-vs-that/)
```suggestion
  A valid array must be an array that contains either a list of strings or arrays of strings and can mix both (e.g. `["director:Mati Diop", ["genre:Comedy", "genre:Romance"]]`).
```

Reply:
great info thanks